README.md

```diff
@@ -51,7 +51,7 @@ func main() {
 		for _, path := range paths {
 			path = strings.TrimSuffix(path, "/")
 			path = strings.TrimSuffix(path, "\\")
-			hash := mozillainstallhash.GetMozillaInstallHash(path)
+			hash := mozillainstallhash.MozillaInstallHash(path)
```

Comment:
The return type has also changed: the function returns an `error` as well now.

Reply:
Nice catch. I'm actually rewriting the README at the moment so I'll be sure to change this.